### ISSUE TITLE
[Question] What value should I provide to `environment`?

### ISSUE THREAD

**Issue by ericgrubaughcompass** [CircleCI-Public/jira-orb#17]
I don't understand what the `environment` string should be.

Using this code:
```yaml
      - jira/notify:
          pipeline_id: << pipeline.id >>
          pipeline_number: << pipeline.number >>
          environment: 'sb1'
          environment_type: 'development'
          job_type: deployment
```
The resulting Deployment looks like:

![image](https://github.com/CircleCI-Public/jira-orb/assets/96068756/48899545-5c50-42d0-8d1c-968a71c623d0)

I was expecting the display name of the Environment to be `'sb1'`, but looking at the Orb source, it places my `environment` value in the `id` property of an `environment`. What ID is expected here? Where are these Environments/IDs defined?

**Comment by kardashov** [comment]
I have the same problem. 
What is displayed in Jira's UI is  _environment.displayName_ field. It is populated by orb like this **"displayName": "${CIRCLE_JOB}::${JIRA_VAL_PIPELINE_ID}"**
here [deployment_payload.json](https://github.com/CircleCI-Public/jira-orb/blob/main/src/scripts/deployment_payload.json#L29 ).

According to example in [Jira API reference](https://developer.atlassian.com/cloud/jira/software/rest/api-group-deployments/#api-rest-deployments-0-1-bulk-post) _environment.displayName_ should have human readable form, not identifiers:
![image](https://github.com/CircleCI-Public/jira-orb/assets/5932404/8cfd1c6b-c5a2-4c2e-9ca2-19b3ccec3fa2)

I think it will be better to populate BOTH _environment.id_ and _environment.displayName_ fields with user provided _environment_ parameter. Or just switch definitions of id and displayName in [deployment_payload.json](https://github.com/CircleCI-Public/jira-orb/blob/main/src/scripts/deployment_payload.json#L28-L29)

**Comment by mattrabe** [comment]
I've used @kardashov's observations to create the PR above

**Comment by mattrabe** [comment]
While we await a fix merged into this orb, I've published a fork of v2.0.1 of this orb, but with my PR change applied and v2.0.2 assigned: https://circleci.com/developer/orbs/orb/fpcs/jira?version=2.0.2
